engine/components-api/src/main/java/com/cloud/storage/StorageManager.java

```diff
@@ -106,7 +106,14 @@
      * @param poolId
      * @return comma separated list of tags
      */
-    public String getStoragePoolTags(long poolId);
+    String getStoragePoolTags(long poolId);
+
+    /**
+     * Returns a list of Strings with tags for the specified storage pool
+     * @param poolId
```

Comment:
If you do not have anything to say about a parameter, you do not need to declare it here.

Reply:
I think when a name is self explanatory don't explain. You'd run the risk of confusing. However when making javadoc, do include all parameters, also to not confuse things. milage and religious views may vary